src/main/java/core/mvc/JsonView.java

```diff
@@ -21,14 +21,16 @@ public void render(Map<String, ?> model, HttpServletRequest request, HttpServlet
         response.setCharacterEncoding(StandardCharsets.UTF_8.name());
         ServletOutputStream outputStream = response.getOutputStream();
 
+        if(model.size() == 0) {
+            return;
+        }
+
         if (model.size() > 1) {
             jsonMessageConverter.write(outputStream, model);
             return;
         }
 
-        Optional<String> key = model.keySet().stream().findFirst();
-        if (key.isPresent()) {
-            jsonMessageConverter.write(outputStream, model.get(key.get()));
-        }
+        String key = model.keySet().stream().findFirst().get();
+        jsonMessageConverter.write(outputStream, model.get(key));
```

Comment:
size 1인 경우만이라면 아래와 같이 써도 되겠네요 :)
```suggestion
        jsonMessageConverter.write(outputStream, model.values().iterator().next());
```